packages/api/src/routes/lightclient.ts

```diff
@@ -1,52 +1,54 @@
-import {Path} from "@chainsafe/ssz";
+import {ContainerType, Path, VectorType} from "@chainsafe/ssz";
 import {Proof} from "@chainsafe/persistent-merkle-tree";
-import {altair, ssz, SyncPeriod} from "@chainsafe/lodestar-types";
-import {
-  ArrayOf,
-  reqEmpty,
-  ReturnTypes,
-  RoutesData,
-  Schema,
-  sameType,
-  ContainerData,
-  ReqSerializers,
-  ReqEmpty,
-} from "../utils";
+import {altair, phase0, ssz, SyncPeriod} from "@chainsafe/lodestar-types";
+import {ArrayOf, ReturnTypes, RoutesData, Schema, sameType, ContainerData, ReqSerializers} from "../utils";
 
 // See /packages/api/src/routes/index.ts for reasoning and instructions to add new routes
 
+export type LightclientSnapshotWithProof = {
+  header: phase0.BeaconBlockHeader;
+  currentSyncCommittee: altair.SyncCommittee;
+  // TODO: Not really necessary since it can be fetched with an update
+  nextSyncCommittee: altair.SyncCommittee;
+  /** Single branch proof from state root to currentSyncCommittee parent */
+  syncCommitteesBranch: Uint8Array[];
+};
```

Comment:
This no longer includes `genesis_time` and `genesis_validators_root` from the previous `init_proof` endpoint, making it difficult to have the client sync its beacon clock.

Reply:
genesis_time and genesis_validators_root should be known ahead of time. For all known networks it should be hardcoded in the lightclient source. For devnets this information can be gathered once by the user through some other means, in a similar way like we distribute config files to join networks.

Reply:
Yeah, but how to validate that the checkpoint is indeed rooted in that particular chain? Or is this assumed implicitly?

Reply:
The user needs to provide a trusted checkpoint, if the block is from another chain the lightclient server won't have any snapshot associated with that block root. Also, updates from a different change won't pass signature validation